index.js

```diff
@@ -11,6 +11,14 @@ module.exports = (robot) => {
 async function handleRelease (robot, context) {
   const owner = context.payload.repository.owner.login
   const repo = context.payload.repository.name
+
+  // log messages with a user/repo prefix
+  function log (msg) {
+    robot.log('%s/%s: %s', owner, repo, msg)
+  }
+
+  log(`${context.payload.release.tag_name} published!`)
```

Comment:
@bkeepers I think that there is `context.log` that logs the owner/repo together with the message, right?